homeassistant/components/shelly/switch.py

```diff
@@ -71,6 +80,23 @@ class BlockSwitchDescription(BlockEntityDescription, SwitchEntityDescription):
     entity_category=EntityCategory.CONFIG,
 )
 
+RPC_SWITCHES = {
+    "switch": RpcSwitchDescription(
+        key="switch",
+        sub_key="output",
+        entity_category=EntityCategory.CONFIG,
```

Comment:
```suggestion
```
This is the main device Switch output and not a config entity

Reply:
Changed PR to introduce script switch only. Rework will be done in a separate PR